src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -1,16 +1,21 @@
-﻿using System;
+﻿#if NET6_0_OR_GREATER
+#define BLAZOR_ENABLED
+#endif
```

Comment:
I must be missing something here. When would `NET6_0_OR_GREATER` ever not be defined? That's all we support here, isn't it?

Reply:
we build this sln in a not .net6 world also for now (legacy to better have tool support)

Reply:
Right, this shared sample is currently used also in non-.NET 6 scenarios, so all Blazor stuff is excluded from such builds. I think this eventually goes away and it's all .NET 6, but for now it needs to stay here to compile.